hack/e2e-chainsaw/_lib/run-kubernetes.sh

```
@@ -441,14 +441,30 @@ EOF
+  # `drbd.linbit.com/lost-quorum` taint delayed the worker DataVolume imports,
+  # the worker VMIs were created ~2m into this wait, and the guests then had
+  # only ~10m to import Talos, boot, register a kubelet and let cilium turn the
+  # nodes Ready. They did not make it: both kubernetes-latest and
+  # kubernetes-previous failed here at exactly 12m with zero Nodes registered
+  # and the tenant cilium HR still mid-install. A less-loaded fleet run passed
+  # the same suites unchanged, so this is load-induced slowness, not a stuck
+  # bring-up; 18m restores margin and still sits well inside the 40m step
+  # timeout (the downstream LB/NFS/ouroboros checks add ~10-15m on the happy
+  # path).
+  if ! timeout 18m bash -c '
     until [ "$(kubectl --kubeconfig tenantkubeconfig-'"${test_name}"' get nodes --no-headers 2>/dev/null | grep -cw Ready)" -ge 2 ]; do
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)